index.html

```diff
@@ -0,0 +1,1117 @@
+                    alt="" class="w-100 h-100 col-md-4 col-lg-3">
+                  <h5>Congressman Mark DeSaulnier in Critical Condition With Pneumonia</h5>
+                </div>
+                <div class="col-lg-9">
+                  <p>The doctors are doing everything they can to care for the Congressman," the representative's
+                    office
+                    said
+                    in a statement.</p>
+                </div>
+              </div>
+            </section>
+            <section id="culture-travel-mobile" class="row d-md-none mt-5 pt-4">
+              <h2 class="col-12">CULTURE & TRAVEL</h2>
+              <div class="col-md-4 col-lg-4">
+                <img
+                  src="https://d.newsweek.com/en/full/1574874/spanish-steps-italy.jpg?w=397&h=397&f=ad0f7bdfd3dc9fffc99ae723d926ac32"
+                  alt="" class="w-100">
+                <h3 class="h3-top-story mt-3">Social Distancing Around the World in Photos</h3>
+                <p>From Rome's desolate Spanish Steps to New York's deserted Times Square, take a look at some of the
+                  world's
+                  busiest landmarks that are all now but eerily empty.</p>
+              </div>
+              <div class="col-md-4 col-lg-4">
+                <img
+                  src="https://d.newsweek.com/en/full/1575422/breakfast-lunch-dinner.webp?w=397&h=397&f=a6dbd65ad3ba41063a6b46b1f92b7a60"
+                  alt="" class="w-100">
+                <h3 class="h3-top-story mt-3">Wanderlust Stricken? The 10 Best Travel TV Shows to Watch Now</h3>
+                <p>Travelers can still explore the world while binge-watching these travel TV shows in home isolation.
+                  Thanks
+                  to streaming services, we can virtually send ourselves to nearly any and every part of the planet in
+                  just a
+                  few clicks.</p>
+              </div>
+              <div class="col-md-4 col-lg-4">
+                <img
+                  src="https://d.newsweek.com/en/full/1574573/taj-mahal.webp?w=397&h=397&f=6059c9040fd468ee41988930ca76f210"
+                  alt="" class="w-100">
+                <h3 class="h3-top-story mt-3">You Can Still See the World With These Virtual Tours</h3>
+                <p>What to do with the free time available when trips to the movies, a museum or the theater are out
+                  of
+                  the
+                  question? Here is a list of museums and other popular tourist attractions that have entire
+                  collections
+                  or
+                  performances available online.</p>
+              </div>
+            </section>
+
+            <section id="latest-news">
+              <h2>Latest news</h2>
+              <div class="col-lg-12 d-flex list-item latest-news border-top mb-2 mb-md-0">
+                <img
```

Comment:
Check the spacing on this section. Your articles are more separated than the original page.

On medium screens, your images lose the proper sizing.
![Screenshot from 2020-04-13 08-29-50](https://user-images.githubusercontent.com/51926937/79124183-62bbdc80-7d61-11ea-9893-f5a0cad6ccb3.png)